lib/fog/openstack/requests/identity_v3/list_role_assignments.rb

```diff
@@ -3,33 +3,39 @@ module Identity
     class OpenStack
       class V3
         class Real
-          def list_role_assignments(options={})
-            params = Hash.new
-            params['group.id'] = options[:group_id] if options[:group_id]
-            params['role.id'] = options[:role_id] if options[:role_id]
-            params['scope.domain.id'] = options[:domain_id] if options[:domain_id]
-            params['scope.project.id'] = options[:project_id] if options[:project_id]
-            params['user.id'] = options[:user_id] if options[:user_id]
-            params['effective'] = options[:effective] if options[:effective]
-
-            params['page'] = options.fetch(:page, 1)
-            params['per_page'] = options.fetch(:per_page, 30)
+          def list_role_assignments(options = {})
+            # Backwards compatibility name mapping
+            name_mapping = {
```

Comment:
For some reason, this one method contains totally different attributes, than rest of keystone methods. 

I wonder if Fog should just offer the single pane of glass here. I could just remove the deprecation and support both standard option names and those that are in API doc. I guess we should be solving such inconsistencies in Fog, or not? It definitely puts more weight on Fog's shoulders. :-)